queue_services/entity-filer/filer_service.py

```diff
@@ -21,26 +21,37 @@
 from entity_filer.resources import register_endpoints
 from entity_filer.worker import APP_CONFIG, FLASK_APP, cb_subscription_handler, flags, qsm
 from entity_queue_common.service_utils import logger
+from structured_logging import StructuredLogging
+
+
+def setup_logger(app, flag_on):
```

Comment:
Current logging info is not displayed in GCP but showns in OCP. Trying to set logger up based on FF.